app/views/LocationShare/services/handleLiveLocationUrl.ts

```
@@ -0,0 +1,109 @@
+			endedIds = new Set();
+		}
+	}
+	return endedIds!;
+}
+async function saveEndedSet() {
+	if (!endedIds) return;
+	try {
+		await AsyncStorage.setItem(ENDED_KEY, JSON.stringify(Array.from(endedIds)));
+	} catch (e) {
+		
+	}
```

ESLint

[error] Empty block statement.

(no-empty)